.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Simple CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3  
+
+    - name: Set up JDK 21
+      uses: actions/setup-java@v3  
+      with:
+        distribution: 'corretto'
+        java-version: '21'  
+
+    - name: Cache Maven packages
+      uses: actions/cache@v3  
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+
+    - name: Build and Test
+      run: mvn clean test 
```

Comment:
![image](https://github.com/user-attachments/assets/ba998f31-7830-4b2b-98c8-69e5f66533ab)
Hi) just update your solution to build jar files)

Reply:
is it just adding "package" at the end?